contracts/contracts/solidity/interfaces/IStarknetMessaging.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0.
```

Comment:
Let's use the submodule now that we have it :)

https://github.com/starkware-libs/starkgate-contracts/blob/main/src/starkware/starknet/solidity/IStarknetMessagingEvents.sol